task/src/test/java/org/exoplatform/platform/qa/ui/task/smoke/LabelManagementTestIT.java

```diff
@@ -0,0 +1,90 @@
+package org.exoplatform.platform.qa.ui.task.smoke;
+
+import com.codeborne.selenide.Condition;
+import org.exoplatform.platform.qa.ui.commons.Base;
+import org.exoplatform.platform.qa.ui.selenium.platform.HomePagePlatform;
+import org.exoplatform.platform.qa.ui.task.pageobject.ManagementLabels;
+import org.exoplatform.platform.qa.ui.task.pageobject.ManagementTasks;
+import org.junit.jupiter.api.BeforeEach;
+import org.junit.jupiter.api.Tag;
+import org.junit.jupiter.api.Test;
+
+import static com.codeborne.selenide.Selectors.byText;
+import static com.codeborne.selenide.Selenide.$;
+import static org.exoplatform.platform.qa.ui.selenium.Utils.getRandomNumber;
+import static org.exoplatform.platform.qa.ui.selenium.locator.taskmanagement.TaskManagementLocator.ELEMENT_TASKS_LIST;
+import static org.exoplatform.platform.qa.ui.selenium.logger.Logger.info;
+
+/**
+ * Created by exo on 16/06/17.
+ */
+@Tag("smoke")
+@Tag("task")
+public class LabelManagementTestIT extends Base {
+
+    ManagementLabels managementLabels;
+
+    HomePagePlatform homePagePlatform;
+
+    ManagementTasks managementTasks;
+
+    @BeforeEach
+    public void setupBeforeMethod() {
+        info("Start setUpBeforeMethod");
+
+        homePagePlatform = new HomePagePlatform(this);
+        managementLabels = new ManagementLabels(this);
+        managementTasks = new ManagementTasks(this);
+    }
+
+    @Test
+    public void test01_Add_Label() {
+        String label = "label" + getRandomNumber();
+        info("add task");
+        homePagePlatform.goToTaskPage();
+        info("Add label");
+        managementLabels.addLabel(label);
+       ;
+        info("delete label");
+        managementLabels.deleteLabel(label);
```

Comment:
There is no check in this test (to test if the label has well been added and deleted), is it intended ? (this remarks is for others tests of this class has well)